src/GovUk.Education.ExploreEducationStatistics.Publisher/Functions/PublishStagedReleaseContentFunction.cs

```diff
@@ -103,7 +108,16 @@ public class PublishStagedReleaseContentFunction
 
                     await _contentService.DeletePreviousVersionsDownloadFiles(releaseIds);
                     await _contentService.DeletePreviousVersionsContent(releaseIds);
-                    await _notificationsService.NotifySubscribers(releaseIds);
+
+                    var releaseIdsToNotifySubscribersFor = _contentDbContext.Releases
```

Comment:
If you take on my other suggestion then this won't be necessary as NotifySubscribers will change to only send notifications if applicable for all the Publications.